<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterForm</class>
 <widget class="QDialog" name="FilterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>225</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Filtering</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="species">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Species</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="bin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Bin</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Select features to remove:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="comboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>E</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="grade">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Grade</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="institution">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Institution</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QTextEdit" name="institution_text">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="checkBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Match all</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QTextEdit" name="bin_text">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QTextEdit" name="species_text">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FilterForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FilterForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
